node/src/neighborhood/mod.rs

```diff
@@ -5214,6 +5310,54 @@ mod tests {
         assert_eq!(expected_public_keys, actual_keys);
     }
 
+    fn assert_route_query_message(min_hops_count: Hops) {
+        let hops = min_hops_count as usize;
+        let nodes_count = hops + 1;
+        let root_node = make_global_cryptde_node_record(4242, true);
+        let mut nodes = make_node_records(nodes_count as u16);
+        nodes[0] = root_node;
+        let db = linearly_connect_nodes(&nodes);
+        let mut subject = neighborhood_from_nodes(db.root(), nodes.get(1));
+        subject.min_hops_count = min_hops_count;
+        subject.neighborhood_database = db;
+
+        let result = subject.make_round_trip_route(RouteQueryMessage {
+            target_key_opt: None,
+            target_component: Component::ProxyClient,
+            return_component_opt: Some(Component::ProxyServer),
+            payload_size: 10000,
+            hostname_opt: None,
+        });
+
+        let assert_hops = |cryptdes: Vec<CryptDENull>, route: &[CryptData]| {
+            assert_eq!(cryptdes.len(), route.len());
+            for (cryptde, data) in cryptdes.into_iter().zip(route) {
+                decodex::<LiveHop>(&cryptde, data).unwrap();
+            }
+        };
+        let mut route = result.clone().unwrap().route.hops;
+        let route_length = route.len();
+        let _accounting = route.pop();
```

Comment:
Bert would like a comment telling why you're throwing away the last value. He might be satisfied with a better name for `_accounting`, if the name is really, really good.

Reply:
There is an old test, written during the substratum times. It uses the variable name `_accounting` to refer to the last packet that is intended for the Originating Node.

The reason why I used this name is to keep the name consistent with the other test.

I added to new comments to explain what's going on in this test.